srcs/web/WebhookManager.py

```diff
@@ -92,15 +92,19 @@ def workflow_job(self, payload: WebHook):
             }
         )
 
-        runner_m.factory.cloud_manager.update_vm_metadata(
-            payload.workflow_job.runner_name,
-            dict(
-                status=status["status"],
-                repository=payload.repository.name,
-                workflow=payload.workflow_job.workflow_name,
-                job=payload.workflow_job.name
+        if (
+            payload.action != "queued"
+            and "gcloud" in payload.workflow_job.runner_name
```

Comment:
I tried many things, and eventually, I realized that it simply required excluding GitHub runners from executing this function. I added a condition to prevent that. Let me know if you think it's good or if you prefer another approach. I find it quite consistent, and essentially, we want this function to always execute except for GitHub.